javascript/smells/leftover_debugging.yaml

```diff
@@ -0,0 +1,29 @@
+rules:
+  - id: javascript-alert
+    patterns: 
+        - pattern-either: 
+          - pattern: alert()
+          - pattern: alert($X)
```

Comment:
you can use alert(...)
will match 0 or n times.

Reply:
Or do you just want to check for call arities of 0 and 1?

Reply:
Yup, arities 0 and 1